app/tally_handler_test.go

```diff
@@ -190,6 +191,7 @@ func (suite *tallyHandlerSuite) TestTallyOutcomes() {
 		proposal := suite.createProposal()
 
 		// no stake
+		suite.app.DeleteGenesisValidator(suite.T(), suite.ctx)
```

Comment:
Total bonded tokens becomes non-zero, which returns true for `burns`.

Could be removed + expected value changed, but thought it would be more clear if we have a clean no-validator state for the test.